Dockerfile

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM centos:centos7
+LABEL maintainer="Presto community <https://prestosql.io/community.html>"
+
+ENV JAVA_HOME /usr/lib/jvm/java-11
+RUN \
+    set -xeu && \
+    yum -y update && \
+    yum -y install java-11-openjdk-devel less && \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    rm -rf /tmp/* /var/tmp/*
+
+RUN \
+    set -xeu && \
+    mkdir -p /data/presto
+
+ARG PRESTO_VERSION
+COPY presto-server/target/presto-server-${PRESTO_VERSION} /usr/lib/presto/
```

Comment:
This target directory has a trailing slash but the `bin` and `etc` below do not. I assume it doesn't matter, but we should make them consistent.

Reply:
👍